src/constants/endpoints.ts

```diff
@@ -5,6 +5,12 @@ export const API_ENDPOINT_URLS = {
   [BscChainId.TESTNET]: 'https://testnetapi.venus.io/api',
 };
 
+export const getApiEndpointUrl = (chainId: BscChainId, isOnPreview = false) => {
+  if (isOnPreview) return 'https://api-preview.venus.io/api';
+
+  return API_ENDPOINT_URLS[chainId];
+};
```

Comment:
Can be either this, or changing the `API_ENDPOINT_URLS` to use the new env var as the key, instead of the `chainId`, since preview and mainnet have the same `chainId`

Reply:
I would refactor the API_ENDPOINT_URLs because they will need to get refactored if we launch on other networks

Reply:
I second Corey. If we launch on other networks, we'll very likely provide a chain ID to the API endpoint so it knows what data to load but its URL would be the same for each environment.

Reply:
Done, thanks guys!